decidim-accountability/app/controllers/decidim/accountability/admin/results_controller.rb

```diff
@@ -56,8 +56,32 @@ def destroy
           redirect_to results_path(parent_id: result.parent_id)
         end
 
+        def proposals
+          respond_to do |format|
```

Comment:
Looks like the specs are not hitting this method. Is it difficult to test as a `system` spec? If so, we should at least unit test it. It looks fairly complex and prone to errors.